src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let mens;
+
+  century
+    ? mens = people.filter(man => man.sex === 'm'
+    && century === Math.ceil(man.died / 100))
+    : mens = people.filter(man => man.sex === 'm');
```

Comment:
Move condition inside filter, fix in all cases

Reply:
fixed